missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+			country_event = ULM_zael_event.3
+			add_mercantilism = 10
+			hidden_effect = {
+				limit = { guaranteed_by = emperor }
+				add_imperial_influence = 20
+			}
+		}
+	}
+	
+	ULM_defensive_overhaul = {
+		icon = mission_conquer_50_development.png
+		required_missions = { ULM_swabians_and_the_swiss ULM_establish_milf }
+		position = 5
+		
+		trigger = {
+			OR = { 
+				any_ally = {
+					primary_culture = swabian
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_tradition = 70
+				}
+				any_ally = {
+					primary_culture = swiss
+					has_opinion = {
+						who = ROOT
+						value = 150
+					}
+					army_tradition = 70
+				}
+				any_country = {
+					NOT = {
+						OR = {
+							primary_culture = swiss
+							primary_cultre = swabian
+						}
+					}
+				}
+			}
+            1872 = {
+				type = all
+				OR = {
+					has_building = fort_15th
+					has_building = fort_16th
+					has_building = fort_17th
+					has_building = fort_18th
+				}
+            }
+			treasury = 1000
+		}
+		
+		effect = {
+			treasury = -600
+			1872 = {
+				add_permanent_province_modifier = {
+					name = ULM_defensive_overhaul
+					duration = -1
+				}
+			}
+			custom_tooltip = emp_mlo_fortify_alps_effects_tooltip
+			hidden_effect = {
+				every_owned_province = {
+					limit = {
+						province = 1872
+					}
+					if = {
+						limit = { has_building = fort_17th }
+						remove_building = fort_17th
+						add_building = fort_18th
+					}
+					else_if = {
+						limit = { has_building = fort_16th }
+						remove_building = fort_16th
+						add_building = fort_17th
+					}
+					else_if = {
+						limit = { has_building = fort_15th }
+						remove_building = fort_15th
+						add_building = fort_16th
+					}
+				}
+			}
+		}
+	}
+}
+
+ULM_province_development = {
+	slot = 3
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+	
+	ULM_a_free_city = {
+		icon = mission_early_game_buildings.png
+		required_missions = { }
+		position = 1
+		
+		trigger = {
+			1872 = {
+				num_of_times_improved = 3
+				has_building = temple
+			}
+		}
+		
+		effect = {
+			add_prestige = 10
+			1872 = {
+				add_permanent_province_modifier = {
+					name = ULM_basic_improvements
+				}
+			}
+		}
+	}
+	
+	ULM_early_municipal_safeguards = {
+		icon = mission_colonial.png
+		required_missions = { ULM_a_free_city }
+		position = 2
+		
+		trigger = {
+			army_size_percentage = 1
+			treasury = 100
+		}
+		
+		effect = {
+			treasury = -50
+			1872 = {
+				add_building = fort_15th
+				add_permanent_province_modifier = {
+					name = ULM_defensive_safeguards
+				}
+			}
+		}
+	}
+	
+	ULM_hundred_year_plans = {
+		icon =
+		required_missions = { ULM_early_municipal_safeguards }
+		position = 3
+		
+		trigger = {
+		
+		}
+		
+		effect = {
+		
+		}
+	}
+	
+	ULM_establish_milf = {
+		icon = 
+		required_missions = { ULM_imperial_bankers ULM_hundred_year_plans ULM_scriveners }
+		position = 4
+		
+		trigger = {
+		
```

Comment:
What I do if I can't/don't feel like writing a low return-on-investement mission, localisation, modifier, etc. I write
```
#TODO
```
So that I can CTRL+F the entire directory and finish when I have time, instead of leaving it blank.

Reply:
Just wanted to wrap this up for the night